_posts/2017/2017-11-09-devoxx-belguim-day-2.md

```diff
@@ -0,0 +1,88 @@
+authors: ["Dirk J."]
+---
+
+Second day.
+It will be hard to top the first day, but the agenda is promising again.
+Read on what my schedule looked like on Thursday, and what the talks were like (imho).
+
+## Lazy Java
+
+To start the day, [Mario Fusco](https://twitter.com/mariofusco) talked about how you could introduce more lazy 
+evaluation in your Java code, mainly by using lambdas.
+While the content was , the presentation was 
+
+
+## gRPC vs REST: let the battle begin!
+
+With [Alex Borysov](https://twitter.com/aiborisov) being the gRPC advocate and 
+[Mykyta Protsenko](https://twitter.com/mykyta_p) taking the REST part, this was a good session. 
+You could learn about the main characteristics of both styles, and where each of them have their strengths and 
+weaknesses. Worth watching.
+
+## Going Reactive with Spring 5 & Project Reactor
+
+Seeing [Josh Long](https://twitter.com/starbuxman) giving a presentation is always a highlight event, but this time he
+was supported by [Mark Heckler](https://twitter.com/MkHeck).
+Together they gave a really amazing introduction to reactive Spring.
+This talk is highly recommended, and you might want to watch it even when not interested in the content, just for the
+pure joy of watching a fantastic live coding show!
+
+## Quickie: SOLID Principles for Cloud Native Containers
+
+While [Bilgin Ibryam](https://twitter.com/bibryam) showed nothing fundamentally new, he very nicely summarized a lot
+of best practices and good advices for working with Cloud Naitve containers in this 15 minutes quickie. Recommended! 
+
+## Quickie: Ally for Women in Technology
+
+In this quickie, [Heather VanCura](https://twitter.com/heathervc) presented her top 10 things (mostly) man can do to
+make the IT world a better place to work in by making it more welcoming to women.
+IMHO, most of those top 10 should be mandatory to everyone, rno matter if there are woman in your team or not.
+If you're interested in this topic, these 15 minutes are worth viewing.
+And if not, well... YOU SHOULD BE!
+
+## Spring Boot 2.0 Web Applications
+
+The second Spring talk of the day I visited was an introduction to the new features and paradigms of Spring Boot 2.0,
+given by [Stéphane Nicoll](https://twitter.com/snicoll) and [Brian Clozel](https://twitter.com/bclozel).
+Although this session could not keep up with the Josh Long talk with regards to entertainment, the content was also
+quite good.
+If you havn't had a deeper look at the new possibilities that come with Spring Boot 2.0, I would recommend this as a
+good summary.
+
+## How Events Are Reshaping Modern Systems
+
+Akka inventor and Lightbend CTO [Jonas Bonér](https://twitter.com/jboner) gave his view on how event driven architecture
+solves a lot of problems software systems are facing nowadays.
+Ranging from the general concepts of commands end events, touching actor-based systems (guess which one
+{% emoji simple_smile %}), to event sourcing, a lot of topics were covered.
```

Comment:
The emoji simple_smile doesn't seem to work. Please try just with {% emoji smile %}